storybookjs/storybook
# Modified the Knobs onChange debounce function so that knob storage occurs on keypress but rendering waits until debouncing has completed

Issue: #6005

## What I did
Modified the component force render debounce so that knob storage still occurs on keypress but the force rendering is debounced. In the 5.0.1 version, the debouncing rate is 150ms which is a little faster than the average typing speed. The change in this PR sets the debouncing rate at 325ms so that the component renders when the user has completed their thought. Frameworks render at different speeds and waiting longer reduces glitchiness. I tested this change in the CLI project to ensure that characters were not being removed as seen in Vue projects.

*I chose 325ms because I felt that it was long enough to reduce glitchiness but not too long to cause frustration. If people believe that this time is too long or too short, please comment below so that we can resolve before merging this in. 

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

@shilman @kristremblay @kylepeeler
